src/lib/utils.ts

```diff
@@ -0,0 +1,9 @@
+export const pluralize = (count: number, singular: string, plural: string): string => {
+  return count === 1 ? singular : plural
+}
+
+export const formatErrorMessage = (error: unknown) => {
```

Comment:
A little utility function for rendering error message in the UI so you don't need to write this logic again and again when we are checking errors returned from the server.